client/src/redux/actions.js

```diff
@@ -0,0 +1,13 @@
+export const testAction = () => dispatch => {
```

Comment:
Actions are plain javascript objects. These methods are `action creators` in that they return actions. Actions are intended to represent a change that should happen to the store. They don't actually mutate the store in any way!